dallinger/recruiters.py

```diff
@@ -171,8 +200,19 @@ def close_recruitment(self):
         """
         logger.info("Close recruitment.")
 
+    def _extract_blacklist_from_config(self):
+        # At some point we'll support lists, so all service code supports them,
+        # but the config system only supports strings for now, so we convert:
+        blacklist = self.config.get('blacklist')
+        if blacklist:
+            blacklist = tuple([item.strip() for item in blacklist.split(',')])
+        else:
+            blacklist = ()
+
+        return blacklist
```

Comment:
I don't think it should hold up merging this, but I think we should add a story or ticket for it so we don't forget.